fuse_loss/package.xml

```diff
@@ -11,19 +11,26 @@
   <author email="swilliams@locusrobotics.com">Stephen Williams</author>
   <license>BSD</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
nit [`ament_cmake_ros` instead of `ament_cmake`](https://github.com/locusrobotics/fuse/pull/287/files#diff-c18b20aa084503537399b799cf05fbe3443eff62b48cff23c3c17ecec57fc117R14).

Reply:
https://github.com/locusrobotics/fuse/compare/25e6567ae3925ff56f5db18a4ea8d3dbb8e320f1..7e4da3d38f24cb4f39f0536c33531388fb297692